packages/components/src/components/wizard/wizard.tsx

```diff
@@ -1,42 +1,57 @@
-import PropTypes from 'prop-types';
+import PropTypes, { object } from 'prop-types';
```

Comment:
are you sure in using here props-types?)

Reply:
No. Will remove this